dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.wim.adapter.ldap.fat.ad.ssl/server.xml

```diff
@@ -19,7 +19,7 @@
 		<feature>ldapRegistry-3.0</feature>
 	</featureManager>
 
-	<ldapRegistry id="ldap" realm="SampleLdapADRealm" host="${ldap.server.2.name}" port="${ldap.server.2.ssl.port}" ignoreCase="true"  baseDN="cn=users,dc=secfvt2,dc=austin,dc=ibm,dc=com"
```

Comment:
Why did baseDN change? It should have needed to change if we are running the same test.